app/src/main/java/org/wildaid/ofish/ui/vesseldetails/VesselDetailsFragment.kt

```diff
@@ -104,6 +105,17 @@ class VesselDetailsFragment : Fragment(R.layout.fragment_vessel_details) {
             }
         })
 
+        vessel_details_appbar.addOnOffsetChangedListener(AppBarLayout.OnOffsetChangedListener{
+            _, verticalOffset ->
+            var scrollRange = -1
+            if (scrollRange == -1){
```

Comment:
You do not need this Condition. 
Every time the method is called this is always true